package.json

```diff
@@ -71,7 +71,8 @@
     "chai": "^4.3.4",
     "coveralls": "^3.1.0",
     "cross-env": "^4.0.0",
-    "eslint": "^2 || ^3 || ^4 || ^5 || ^6 || ^7.2.0",
+    "escope": "^3.6.0",
```

Comment:
This change is a workaround to prevent the babel-eslint patch from crashing.

https://github.com/babel/babel-eslint/blob/v8.2.6/lib/patch-eslint-scope.js#L31